src/openforms/variables/tests/test_user_defined_variables.py

```diff
@@ -0,0 +1,56 @@
+from django.test import TestCase
+
+from jsonschema import Draft202012Validator
+
+from ..constants import FormVariableDataTypes
+from ..service import get_json_schema_for_user_defined_variable
+
+
+class UserDefinedVariablesValidJsonSchemaTests(TestCase):
+    validator = Draft202012Validator
+
+    def check_schema(self, properties):
```

Comment:
```suggestion
    def assertValidSchema(self, properties):
```
makes the assertions clear, because I had the impression that the tests were not asserting anything and you had forgotten something :)